DEVELOPER_GUIDE.md

```diff
@@ -74,6 +74,35 @@ Fork [opensearch-project/OpenSearch](https://github.com/opensearch-project/OpenS
 
 ### Install Prerequisites
 
+
+### [Optional] sdkman 
```

Comment:
Add to TOC please.

I think you can make this section "SDKman" to make it consistent capitalization-wise and just say "We recommend a tool such as SDKman to manage JDKs".